config/webpack/entries/main.js

```diff
@@ -1,24 +1,32 @@
 /*
  *  Our main js entrypoint :)
  *
- *  It looks kind of empty because almost everything is happening via webpack.shims.js for now.
- *
  */
 
-import '@/public/dist/uib-templates';
```

Comment:
Does this ever get imported anywhere now? Why do or don't we need it?

Reply:
Those are now imported in `config/client/bootstrap.js`. All the ones ending in `.html` in there are angularjs templates, which get processed through the webpack loaders. Previously there was a gulp task that built the templates into `public/dist/uib-templates`.